packages/cojson/src/tests/sync.storageAsync.test.ts

```diff
@@ -516,13 +520,6 @@ describe("client syncs with a server with storage", () => {
         "storage -> client | CONTENT Map header: true new: After: 7 New: 1",
         "storage -> client | CONTENT Map header: true new: After: 8 New: 1",
         "storage -> client | CONTENT Map header: true new: After: 9 New: 1",
-        "storage -> client | CONTENT Map header: true new: After: 10 New: 0",
```

Comment:
While updating the tests, I've noticed this content message sent with 0 transactions.

It shouldn't be sent, so I've also fixed the bug.